Hardware/JetsonTX2.md

```diff
@@ -22,7 +22,7 @@ The Jetson TX2 is an embedded system-on-module (SOM) developed by NVIDIA.
 <https://elinux.org/Jetson_TX2>
 
 The seL4 kernel has a limited port to the TX2 which supports the SoM
-only in 64-bit mode. SMP, SMMU, Hyp-mode, etc are not supported by the
+only in 64-bit mode. SMP is not supported by the
 current port.
```

Comment:
Turns out SMP is also supported and [tested for TX2](https://github.com/seL4/ci-actions/blob/15610f31ea9d69eacd0e789af36e46f4b3ac23ce/seL4-platforms/platforms.yml#L239-L246)

```suggestion
only in 64-bit mode.
```

Reply:
Cool!